mtags/src/main/scala-3/scala/meta/internal/pc/PcCollector.scala

```diff
@@ -185,6 +221,15 @@ abstract class PcCollector[T](driver: InteractiveDriver, params: OffsetParams):
         def soughtOrOverride(sym: Symbol) =
           sought(sym) || sym.allOverriddenSymbols.exists(sought(_))
 
+        def extensionParam(sym: Symbol) =
```

Comment:
I was bit confused about the function without it.
```suggestion
        def isExtensionParam(sym: Symbol) =
```